web/src/main/resources/application-dev.properties

```
@@ -28,3 +28,17 @@ spring.jpa.hibernate.ddl-auto=create
 
 ma.security.secure-key-token=user1,user2

```

```suggestion
ma.security.secure-key-token=user1,user2
```

Remove these whitespace trails on all the occasions on this file and the other properties files that have this